stable/aws-vpc-cni/templates/cni-metrics-helper.yaml

```diff
@@ -0,0 +1,92 @@
+{{- if .Values.enableMetricsHelper -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cni-metrics-helper
+  labels:
+{{ include "aws-vpc-cni.labels" . | indent 4 }}
+rules:
+  - apiGroups: [""]
+    resources:
+      - nodes
+      - pods
+      - pods/proxy
+      - services
+      - resourcequotas
+      - replicationcontrollers
+      - limitranges
+      - persistentvolumeclaims
+      - persistentvolumes
+      - namespaces
+      - endpoints
+    verbs: ["list", "watch", "get"]
+  - apiGroups: ["extensions"]
+    resources:
+      - daemonsets
+      - deployments
+      - replicasets
+    verbs: ["list", "watch"]
+  - apiGroups: ["apps"]
+    resources:
+      - statefulsets
+    verbs: ["list", "watch"]
+  - apiGroups: ["batch"]
+    resources:
+      - cronjobs
+      - jobs
+    verbs: ["list", "watch"]
+  - apiGroups: ["autoscaling"]
+    resources:
+      - horizontalpodautoscalers
+    verbs: ["list", "watch"]
```

Comment:
The above seems like a very broad ruleset to give to the service account for the metrics helper. Are we sure the above is necessary?

Reply:
it was taken directly from [the upstream yaml manifest](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/config/v1.6/cni-metrics-helper.yaml)